ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -77,5 +79,6 @@ Rectangle {
         font.pixelSize: 10
         readOnly: true
         selectByMouse: true
+        visible: repeatMessageInfo
```

Comment:
This one needs to be revisited too. We probably want to come up with some threshold on when to show the time of a message when there are multiple ones coming from the same user.